src/wmtk/energy/Energy.cpp

```diff
@@ -0,0 +1,9 @@
+#include "Energy.hpp"
+
+Energy::Energy(const Mesh& mesh)
+    : m_mesh(mesh)
+    , m_position_handle(m_mesh.get_attribute_handle<double>("position", PrimitiveType::Vertex)){};
```

Comment:
i think the generic energy interface shouldn't be getting this handle - each energy should request these handles themselves with their constructors? but getting the mesh is fine

Reply:
but the pattern for construction of these individual energies looks fine :+1: